packages/backend/src/services/PromoteService/PromoteService.ts

```diff
@@ -759,7 +759,7 @@ export class PromoteService extends BaseService {
         return space.data;
     }
 
-    private async getOrCreateDashboard(
+    async getOrCreateDashboard(
```

Comment:
If you only want to expose this method for testing purposes we can extend this class.

First, we would need to convert it from `private` to `protected` so it is available in extended classes.

```
class PromoteServiceTest extends PromoteService {
   async getOrCreateDashboard(...) {
       return super.getOrCreateDashboard(...);
   }
}
```